torch_pin.py

```diff
@@ -1,2 +1,2 @@
 TORCH_VERSION = "2.10.0"
-NIGHTLY_VERSION = "dev20250915"
```

Comment:
what's this pin for? if it is specific for ci, maybe we should pin bump et pt pin as well.

Reply:
i didn't follow; if so why et doesn't pin bump as well?